src/components/Table/Table.jsx

```diff
@@ -692,6 +739,23 @@ const Table = (props) => {
               }
             />
           )}
+          {hasAggregations ? (
+            <TableFoot
+              options={{
+                ...pick(
+                  options,
+                  'hasRowSelection',
+                  'hasRowExpansion',
+                  'hasRowActions'
+                ),
+              }}
+              tableState={{
+                aggregations,
+                ordering: view.table.ordering,
+              }}
+              testID={`${id}-table-foot`}
```

Comment:
will want these to be `data-testid`

Reply:
@davidicus Sure thing. Can you explain the differences between when you all use `testID` as shown here: https://github.com/carbon-design-system/carbon-addons-iot-react/blob/next/docs/guides/testing.md#configuration versus using the full `data-testid` prop?

Reply:
Hey Kevin. Sorry, I followed the props down. My initial review I thought you were just adding test id as attribute here. I see in the Table footer component you pass it down to an actual data-testid. You can ignore. Thanks